lib/std/c/linux.zig

```diff
@@ -344,3 +347,15 @@ pub const dirent64 = struct {
     type: u8,
     name: [256]u8,
 };
+
+// Neither glibc nor musl provide a wrapper function for this syscall
+pub const perf_event_open = linux.perf_event_open;
```

Comment:
This is the libc bindings API. If it is not provided by libc, then it is correct to be missing from this file. The best place for this abstraction is in the `std.posix` API layer (status quo).

The posix API layer includes API that is only available on a subset of operating systems, and it's OK if that subset is only one system.

Reply:
The function is available on all Linux systems. The intended way to use it from C code is by manually using the syscall() macro provided by libc, which is equivalent to linux.perf_event_open().

The alternative I see is to add a comptime branch in std.posix which checks if the target os is Linux and then uses linux.perf_event_open() regardless of whether or not libc is linked. I think that's what you're asking me to do?

Reply:
done